lib/coinbase/stake.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Stake operation used for a end-user custody flow.
+  class Stake
```

Comment:
Typically classes are instantializable, have we thought through what it looks like for the managed wallet case?

That will dictate what this class will eventually be